src/dataflow-types/src/client.rs

```diff
@@ -978,7 +1013,7 @@ pub mod partitioned {
             for id in cease.into_iter() {
                 let previous = self.uppers.remove(&(id, instance));
                 if previous.is_none() {
-                    debug!("Protocol error: ceasing frontier tracking for absent identifier {:?} due to command {:?}", id, command);
+                    warn!("Protocol error: ceasing frontier tracking for absent identifier {:?} due to command {:?}", id, command);
```

Comment:
Idk what our norms are here, but this seems to be reporting a bug (a non-fatal error). Is that no longer the case because of reconciliation, for some reason? If that's right, an explainer comment would rule, so that the next person to read this doesn't have the same confusion I had!

Reply:
Oops, leftover from debugging! My bad